crates/proof-of-sql/src/base/database/column_repetition_operation.rs

```diff
@@ -181,4 +203,60 @@ mod tests {
             Column::VarChar((&doubled_strings, &doubled_scalars))
         );
     }
+
+    #[test]
+    fn test_column_repetition_op_varbinary() {
+        let bump = Bump::new();
+
+        let bytes = vec![b"foo".as_ref(), b"bar".as_ref()];
+        let scalars: Vec<TestScalar> = bytes
+            .iter()
+            .map(|b| TestScalar::from_le_bytes_mod_order(b))
+            .collect();
```

Comment:
NIT: this should probably be a hash to reflect what is actually happining in practice.

Or, just do this
```suggestion
        let scalars = vec![TestScalar::from(1), TestScalar::from(2)];
```